### ISSUE TITLE
Cron job woocommerce_cleanup_session not working

### ISSUE THREAD

**Issue by AbdulSamadKt** [woocommerce/woocommerce#19917]
On my live site, the database table wp_woocommerce_sessions is getting too big size and now it is 864 MB. When I check cron jobs using WP Crontrol plugin, I can't find the cron job woocommerce_cleanup_session on the cron events list. 

Is woocommerce_cleanup_session cron job delete data from table wp_woocommerce_sessions? Why woocommerce_cleanup_session cron job not working  / not showing on my website?

**Comment by mikejolley** [comment]
Maybe something unregistered it? Or it was manually deleted?

De-activate and re-activate WooCommerce to get it back.

This will show the following:

![slack - a8c 2018-05-01 12-45-29](https://user-images.githubusercontent.com/90977/39471868-915a453c-4d3d-11e8-9c0d-e43727c43fa9.png)

You can also run cleanup manually via WooCommerce > Status > Tools.

Thanks

**Comment by AbdulSamadKt** [comment]
Can I know how many rows can delete the cron job woocommerce_cleanup_session at a time? Can I increase the number of rows to delete?

**Comment by mikejolley** [comment]
There is no limit.

**Comment by AbdulSamadKt** [comment]
Thank you. I deactivated and activated WooCommerce plugin and the cron job came back!. Now the database is only around 36 MB. 

The root problem was my site shows  “Error establishing a database connection” sometimes for a few seconds. When I checked debug log, it showed "PHP Warning:  mysql_connect(): User nessenti_wp1 already has more than 'max_user_connections' active connections in /home/nessenti/n-essentials.com.au/wp-includes/wp-db.php on line 1633" because the database table wp_woocommerce_sessions was so big.

Thank you @mikejolley
